src/app/qgisapp.cpp

```diff
@@ -14293,6 +14367,42 @@ void QgisApp::toolButtonActionTriggered( QAction *action )
     settings.setEnumValue( QStringLiteral( "UI/defaultVertexTool" ), QgsVertexTool::AllLayers );
   else if ( action == mActionVertexToolActiveLayer )
     settings.setEnumValue( QStringLiteral( "UI/defaultVertexTool" ), QgsVertexTool::ActiveLayer );
+  else if ( action == mActionCircularStringCurvePoint )
+    settings.setValue( QStringLiteral( "UI/defaultCircularString" ), 0 );
```

Comment:
Any reason not to use `setEnumValue`?

Reply:
Yes! I don't know what it is nor how that plays! :thinking:

Reply:
It saves enum names instead of magic numbers.
But sorry, that won't work here, since there are no enum involved.